modules/geopackage/src/lib/parse-geopackage.js

```diff
@@ -381,3 +414,54 @@ function interleaveResults(columns, values) {
 
   return merged;
 }
+
+/**
+ * Get arrow schema
+ *
+ * @param  {Database} db GeoPackage object
+ * @param  {string} tableName  table name
+ * @return {Schema | null} Arrow-like Schema
+ */
+function getArrowSchema(db, tableName) {
+  const metadata = db.exec(
+    `SELECT * FROM \`sqlite_master\` WHERE type='table' AND tbl_name='${tableName}';`
+  )[0];
+  const sql =
+    (metadata.values &&
+      metadata.values[0] &&
+      metadata.values[0][4] &&
+      metadata.values[0][4].toString()) ||
+    '';
```

Comment:
Might be worth a comment to say what `metadata.values[0][4]` is. The name `sql` doesn't mean much to me.